File: add.php
<?php
function sum($a, $b) 
{ 
    return $a + $b; 
}

echo "sum(1,2): " . sum(1,2);
echo "sum(1,(2,3)): " . sum(1,(2,3));
echo "sum((1,2)): " . sum((1,2));

File: add.exs
defmodule Adder do
  def sum(a,b) do
    a + b
  end
end

IO.puts "sum(1,2): #{sum(1,2)}";
IO.puts "sum(1,(2,3)): #{sum(1,(2,3))}";
IO.puts "sum((1,2)): #{sum((1,2))}";
